### ISSUE TITLE
No Typescript type declarations

### ISSUE THREAD

**Issue by SleepyGrump** [saadeghi/theme-change#10]
npm i --save-dev @types/theme-change doesn't work either, thought I'd let you know.

**Comment by Siumauricio** [comment]
to avoid the error i did the following 
 const {themeChange} = require('theme-change');

   useEffect(() => {
      themeChange(false);
   }, []);

**Comment by daison12006013** [comment]
LOL, you can use it this way, to simplify my code without localStorage etc

`window.document.documentElement.setAttribute("data-theme", "cmyk")`

e.g: for svelte

```
onMount(() => {
  window.document.documentElement.setAttribute("data-theme", "cmyk")
});
```

Replace that `cmyk` into your default theme, use $store to create a toggle [similar to this](https://github.com/daison12006013/sveltekit-starter/blob/develop/src/stores/menus.ts#L21)

**Comment by ZingerLittleBee** [comment]
you can view this issue
https://github.com/saadeghi/theme-change/issues/4#issuecomment-920019712

add `declare module 'theme-change'` in `src/xxx.d.ts` to avoid this warning

**Comment by saadeghi** [comment]
Fixed in `2.1.0`
